test/integration/02-user-wallet/02-send-lightning.spec.ts

```diff
@@ -417,12 +418,21 @@ describe("UserWallet - Lightning Pay", () => {
           tokens: amountInvoice,
         })
         await fn(userWallet1)({ invoice: request })
-        const intermediateBalance = await userWallet1.getBalances()
+        const intermediateBalanceSats = await Wallets.getBalanceForWallet({
```

Comment:
why dont you use the helper here??

Reply:
Good catch! This was a remnant from when we returned an object from the function and needed to compare it with a `toBeStrictEqual` at these 3 places in the tests